Header header
string robot_name
int8 robot_id
geometry_msgs/Pose2D robot_pos
geometry_msgs/Point ball_local
uint8 robot_status
uint8 ball_status
uint8 is_moving
